paint-prototype/Assets/Fork/Tom/Scripts/PlayerCamera.cs

```diff
@@ -0,0 +1,45 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerCamera : MonoBehaviour
+{
+    [SerializeField] private float sensitivityX;
+    [SerializeField] private float sensitivityY;
+
+    Camera cam;
+
+    float mouseX;
```

Comment:
Because these two variables are not being used outside the function, they should be declared and limit to its own function scope.
Also, can't we use a `Vector2` here, it could make it cleaner, but `float` is alright.